02-path-working-with-clusters/202-service-mesh/readme.adoc

```diff
@@ -34,7 +34,7 @@ In this exercise we'll look at a few of the features linkerd provides, such as:
 
 === Create a Kubernetes cluster
 
-In order to perform exercises in this chapter, you'll need to deploy configurations to an EKS cluster.  To create an EKS cluster, use the link:../../01-path-basics/102-your-first-cluster#create-a-kubernetes-cluster-with-eks[AWS CLI] (recommended), or alternatively, link:../../01-path-basics/102-your-first-cluster#alternative-create-a-kubernetes-cluster-with-kops[kops].
+In order to perform exercises in this chapter, you'll need to deploy configurations to an EKS cluster. To create an EKS cluster, use the link:../../01-path-basics/102-your-first-cluster#create-a-kubernetes-cluster-with-eks[AWS CLI] (recommended), or alternatively, link:../../01-path-basics/102-your-first-cluster#alternative-create-a-kubernetes-cluster-with-kops[kops].
```

Comment:
The wording does not seem right. EKS cluster can only be created using AWS CLI, not using kops.